core/jobs.py

```diff
@@ -932,7 +941,10 @@ def validate(cls, unused_mapper_spec):
 
 
 class BaseMapReduceJobManagerForContinuousComputations(BaseMapReduceJobManager):
-
+    """Continuous Computation jobs which runs continuously, are used to perform
```

Comment:
English: Continuous computation jobs, which run continuously, are used to perform ...

Reply:
done.